web/src/features/dashboard/components/ScoresTable.tsx

```diff
@@ -131,18 +189,23 @@ export const ScoresTable = ({
           <RightAlignedCell key="one">1</RightAlignedCell>,
         ]}
         rows={data.map((item, i) => [
-          item.scoreName,
+          <LeftAlignedCell
+            key={`${i}-name`}
+          >{`${getScoreDataTypeIcon(item.scoreDataType)} ${item.scoreName} (${item.scoreSource.toLowerCase()})`}</LeftAlignedCell>,
```

Comment:
Opted to keep score label cohesive across charts/tables for now, can look into improving UI feel of the score source display in the future for charts (and potentially across platform), but felt more cohesive this way

Reply:
looks good to me!